lib/master/src/helpers.cc

```diff
@@ -23,23 +34,80 @@ namespace kickcat
             return; // No interactive choice asked: nothing to do
         }
 
-        auto netIntefaces = listInterfaces();
-        for (std::size_t i = 0; i < netIntefaces.size(); ++i)
+        auto net_interfaces = listInterfaces();
```

Comment:
I feel the code style is starting to get inconsistent. I suggest we add a code format checker like `clang-format` what do you guys think? I create an issue if you guys think it's a good idea.